addons/editor-dark-mode/experimental_editor.css

```diff
@@ -1,215 +1,548 @@
+.camera-modal_body_3GKi0,
+.record-modal_body_2VO4l {
+  background-color: var(--editorDarkMode-accent);
+  color: var(--editorDarkMode-accent-text);
 }
-.scratchCommentTextarea::placeholder {
-  color: rgba(255, 255, 255, 0.5);
-}
-.scratchCommentText {
-  fill: #fff;
-}
-.stage_stage_1fD7k {
-  border-color: rgba(255, 255, 255, 0.15);
-}
-.sprite-selector-item_sprite-info_-I0i_,
 .stage-selector_header-title_33xCt,
 .stage-selector_label_1MCfr,
 .stage-selector_count_2QK7D,
 .color-button_color-button-arrow_2NxGY,
-.color-picker_row-header_173LQ,
 .sound-editor_tool-button_2iNn9,
 .sound-editor_effect-button_2zuzT,
-.blocklyWidgetDiv .goog-menuitem,
-.filter_filter-input_1iiEt,
-.filter_filter-input_1iiEt::placeholder,
 .library-item_featured-extension-metadata_3D8E8,
-.monitor_list-index_7tKdl,
-.question_question-label_1tRo2 {
-  color: #fff;
+.question_question-label_1tRo2,
+.camera-modal_disabled-text_G3r68,
+.camera-modal_loading-text_e5804,
+.record-modal_help-text_Jevsk {
+  color: var(--editorDarkMode-accent-text);
 }
-.scratchCategoryMenuItem.categorySelected {
-  background: #333;
+.camera-modal_main-button_1G6x1:disabled {
+  background-color: var(--editorDarkMode-accent-text);
 }
-.blocklyFlyoutBackground {
-  fill: #333;
+.blocklyFlyoutButton:hover {
+  fill: var(--editorDarkMode-accent);
 }
-.blocklyFlyoutLabelText,
-.blocklyFlyoutButton .blocklyText {
-  fill: #fff;
+.blocklyFlyoutButton:hover .blocklyText {
+  fill: var(--editorDarkMode-accent-text);
 }
-.blocklyFlyoutButtonBackground,
-.custom-procedures_workspace_1d2uW .blocklyMainBackground,
-.blocklyFlyoutCheckbox {
-  stroke: #444;
-}
-.blocklyFlyoutButton:hover,
-.blocklyScrollbarHandle {
-  fill: #444;
-}
-.blocklyFlyoutCheckbox {
-  fill: #282828;
-}
-.blocklyBlockCanvas g:not(.checked) .blocklyFlyoutCheckboxPath {
-  stroke: #282828;
+.selector_new-buttons_2qHDd::before {
+  background: linear-gradient(var(--editorDarkMode-accent-opacity0), var(--editorDarkMode-accent));
 }
-.backpack_backpack-item_hwqzQ img {
-  mix-blend-mode: normal;
+.s3devDD > li.boolean::before {
+  border-top: 9px solid var(--editorDarkMode-accent);
+  border-bottom: 10px solid var(--editorDarkMode-accent);
 }
-.sprite-selector_sprite-selector_2KgCX,
-.modal_modal-content_1h3ll.modal_full-screen_FA4cr,
-.library_library-scroll-grid_1jyXm {
-  background-color: #030b16;
+.s3devDD > li.boolean::after {
+  border-top: 9px solid var(--editorDarkMode-accent);
+  border-bottom: 10px solid var(--editorDarkMode-accent);
 }
 .stage-header_stage-button-icon_3zzFK,
 .sprite-info_icon-wrapper_3Wbqq:not(.sprite-info_radio_v-fgn),
+.sprite-info_radio_v-fgn:not(.sprite-info_is-active_ewF6e) img,
 img.tool-select-base_tool-select-icon_tJ-rr,
 .paint-editor_button-group-button-icon_10kVn,
 img[src="/static/assets/7bd7487b704797cb5ab3cb441486ea70.svg"],
 .library-item_featured-extension-metadata-detail_1M9BV img {
-  filter: brightness(0) invert(1);
+  filter: var(--editorDarkMode-accent-filter);
```

Comment:
This mostly works, but there should be a different rule for toggle switches. For example, there isn't appropriate contrast here:

!["show" toggle with the "shown" option selected](https://user-images.githubusercontent.com/9948030/111079624-7c1d5980-84d9-11eb-95b4-6f8face44481.png)

The option which isn't selected should be dimmer. (Probably a different filter should be applied.)

Rules that this applies to:
- `sprite-info_radio_v-fgn`
- `stage-header_stage-button-icon`
- `tool-select-base_tool-select-icon` (unselected paint tools should be dim)
- Possibly ` .paint-editor_button-group-button-icon`? I'm not sure where this shows up.

Reply:
I think adding `opacity(0.7)` to the filter for unselected toggle buttons would work well.

Scratch doesn't make unselected paint tools dim and I don't think dark mode should change that. `paint-editor_button-group-button-icon` are zoom button icons.

Reply:
Fixed.